actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadBackGroundJobActor.java

```diff
@@ -1040,7 +1026,14 @@ private void processUserInfo(
                 ssoManager.removeUser(userMap);
               }
             }
-
+            try {
+              // update the user external identity data
+              Util.addUserExtId(userMap);
+            } catch (Exception ex) {
+              ProjectLogger.log("Exception occurred while updating user external identity", ex);
```

Comment:
From:
"Exception occurred while updating user external identity"

To:
"BulkUploadBackGroundJobActor:processUserInfo: Exception occurred while adding user external identity."